client/layout/index.jsx

```diff
@@ -106,6 +100,11 @@ class Layout extends Component {
 	}
 
 	componentDidUpdate( prevProps ) {
+		// This code should be removed when the nav-unification project has been rolled out to 100% of the customers.
+		if ( config.isEnabled( 'nav-unification' ) ) {
+			window.addEventListener( 'scroll', scrollCallback );
+			window.addEventListener( 'resize', scrollCallback );
+		}
```

Comment:
@frontdevde Just checking...this logic was moved into `componentDidUpdate` but that method is 

> ...not called for the initial render.

Should we have in both places?

Incidentally, this is why hooks like `useEffect` are a improvement over lifecycle methods.

Reply:
The root of the issue is that `config.isEnabled( 'nav-unification' )` returns a false negative at mount time which resulted in this never being triggered. `componentDidUpdate` on the layout component triggers several times when the app initialises, so we should be fine.